catalog/dags/data_refresh/remap_table_constraints.py

```diff
@@ -0,0 +1,233 @@
+"""
+# Remap Table Constraints
+
+This file contains TaskGroups related to remapping the table constraints from the live media table onto the temp table to prepare it for promotion.
+
+"""
+
+import logging
+from typing import NamedTuple
+
+from airflow.decorators import task, task_group
+
+from common.sql import run_sql
+from data_refresh import queries
+
+
+logger = logging.getLogger(__name__)
+
+
+class ConstraintInfo(NamedTuple):
+    constraint_table: str
+    constraint_name: str
+    constraint_statement: str
```

Comment:
Nit: since these are already attached to an object with `Constraint` in the name (and will likely be referenced by a variable named `constraints` or similar, could these maybe just be `table`, `name`, and `statement`?